docs/Migrating to Newer SDL Java Suite Versions/Updating to v5.6/index.md

```diff
@@ -0,0 +1,116 @@
+# Upgrading to 5.6
```

Comment:
I think this should be "Updating" instead of "Upgrading" so that it matches the name of the page. It looks like we first started using "Upgrading" on `Updating to v4.9` so should change the title there (from "Upgrading" to "Updating") and on each following version.